crates/ignore/src/default_types.rs

```diff
@@ -16,7 +16,7 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("asciidoc", &["*.adoc", "*.asc", "*.asciidoc"]),
     ("asm", &["*.asm", "*.s", "*.S"]),
     ("asp", &[
-        "*.aspx", "*.aspx.cs", "*.aspx.vb", "*.ascx", "*.ascx.cs", "*.ascx.vb",
+        "*.aspx", "*.aspx.cs", "*.aspx.vb", "*.ascx", "*.ascx.cs", "*.ascx.vb", "*.asp"
```

Comment:
Please follow the style of the surrounding code and make sure lines are no longer than 79 columns.

(It's hard to tell if this is longer than 79 columns in the GitHub UI, but it looks like it.)

Reply:
Thank you, I was commit the change from Github online editor directly due to simple change. Forgot to check the line length, it's far exceed 79 already.
Now, fixed.